core/samples/reflection/basic/page.md

```diff
@@ -0,0 +1,49 @@
+# Reflection {#examples-core-reflection-basic}
+
+@brief Using the reflection system.
+
+Lets say you have a type `Person`, which you want to be able to reflect. You
```

Comment:
I'd replace the "reflect" in "you want to be able to reflect" with an use case of reflection.